kubernetes-full.yml

```diff
@@ -0,0 +1,293 @@
+                name: cypress-s3-secrets
+                key: AWS_ACCESS_KEY_ID
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+              secretKeyRef:
+                name: cypress-s3-secrets
+                key: AWS_SECRET_ACCESS_KEY
+        image: agoldis/sorry-cypress-director:cypress-v5
+        imagePullPolicy: "Always"
+        name: director
+        ports:
+        - containerPort: 1234
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 1234
+          periodSeconds: 10
+          timeoutSeconds: 5
+          successThreshold: 2
+          failureThreshold: 5
+        resources:
+          requests:
+            memory: "128M"
+            cpu: 0.2
+          limits:
+            memory: "512M"
+            cpu: 0.5
+      restartPolicy: Always
+      serviceAccountName: ""
+      volumes: null
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dashboard
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dashboard
+  template:
+    metadata:
+      labels:
+        app: dashboard
+      name: dashboard
+    spec:
+      containers:
+      - env:
+        - name: GRAPHQL_SCHEMA_URL
+          value: [YOUR-API-URL-HERE]
+        image: agoldis/sorry-cypress-dashboard:cypress-v5
+        imagePullPolicy: "Always"
+        name: dashboard
+        ports:
+        - containerPort: 8080
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 8080
+          periodSeconds: 10
+          timeoutSeconds: 5
+          successThreshold: 2
+          failureThreshold: 5
+        resources:
+          requests:
+            memory: "128M"
+            cpu: 0.2
+          limits:
+            memory: "512M"
+            cpu: 0.5
+      restartPolicy: Always
+      serviceAccountName: ""
+      volumes: null
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: dashboard-service
+spec:
+  ports:
+  - name: "8080"
+    port: 8080
+    targetPort: 8080
+  selector:
+    app: dashboard
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: api-service
+spec:
+  ports:
+  - name: "4000"
+    port: 4000
+    targetPort: 4000
+  selector:
+    app: api
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mongo-service
+spec:
+  ports:
+  - name: "27017"
+    port: 27017
+    targetPort: 27017
+  selector:
+    app: mongo
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: director-service
+spec:
+  ports:
+  - name: "1234"
+    port: 1234
+    targetPort: 1234
+  selector:
+    app: director
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: cypress-ingresses
+spec:
+  rules:
+  - host: [YOUR-DASHBOARD-URL-HERE]
+    http:
+      paths:
+      - backend:
+          serviceName: dashboard-service
+          servicePort: 8080
+  - host: [YOUR-API-URL-HERE]
+    http:
+      paths:
+      - backend:
+          serviceName: api-service
+          servicePort: 4000
+  - host: [YOUR-DIRECTOR-URL-HERE]
```

Comment:
Sorry if I am stupid, does it allow to access "internal" services using the URLs in `[]`?

Reply:
This is essentially what creates the url, provisioning a loadbalancer and even possibly the DNS entries (if you have k8s set up to do it).